packages/block-library/src/embed/embed-preview.js

```diff
@@ -45,6 +45,7 @@ const EmbedPreview = ( props ) => {
 				scripts={ scripts }
 				title={ iframeTitle }
 				type={ sandboxClassnames }
+				interactive={ interactive }
```

Comment:
Out of curiosity, is there any reason we couldn't use `Disabled` for this?

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/disabled